crates/next-core/src/next_config.rs

```diff
@@ -82,7 +82,7 @@ pub struct NextConfig {
     server_runtime_config: IndexMap<String, serde_json::Value>,
     static_page_generation_timeout: f64,
     swc_minify: bool,
-    target: String,
+    target: Option<String>,
```

Comment:
Looking at the other options it seems this is the way to make it optional, if not let me know 👍